applications/frontend/src/components/header/Banner.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+
+import { UserInfo } from '../../types/user';
+
+const useStyles = makeStyles((theme) => ({
+  mainFeaturedPost: {
+    position: 'relative',
+    backgroundColor: theme.palette.grey[800],
+    color: theme.palette.common.white,
+    margin: theme.spacing(1),
+    backgroundImage: 'url(images/banner.png)',
```

Comment:
this url may be unstable: better use the webpack loader

Reply:
hmm not sure if that's better

Reply:
OK not fundamental changing this now, would like to hear your ideas about asset management.